browser/src/layer/tile/CanvasTileWorker.js.m4

```diff
@@ -0,0 +1,14 @@
+m4_changequote([,])m4_dnl
+m4_dnl# foreachq(x, `item_1, item_2, ..., item_n', stmt)
+m4_dnl# quoted list, alternate improved version
+m4_define([m4_foreachq],[m4_ifelse([$2],[],[],[m4_pushdef([$1])_$0([$1],[$3],[],$2)m4_popdef([$1])])])m4_dnl
+m4_define([_m4_foreachq],[m4_ifelse([$#],[3],[],[m4_define([$1],[$4])$2[]$0([$1],[$2],m4_shift(m4_shift(m4_shift($@))))])])m4_dnl
+m4_define([m4_trim],[m4_patsubst([$1],[^. ?\(.*\) .$])])m4_dnl
+m4_dnl
+// This comment required to work around preprocessor bugs
```

Comment:
If we add another line of
// eslint-disable no-unused-vars
here then that disable the no-unused-vars error of "  3:285  warning  'a' is assigned a value but never used  no-unused-vars" seen in ci